.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{

```

Comment:
do we need vscode specific json in the repo?

Reply:
it's just a convenience for those who use VSCode.